src/cache/lru-cache.ts

```diff
@@ -13,6 +13,8 @@
  */
 export class LRUCache implements Map<string, string> {
   private readonly cache = new Map<string, string>();
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   [Symbol.toStringTag]: string;
```

Comment:
This isn't hurting anything by not being initialized and I didn't want to alter the behavior of a core utility class, so I opted to instead use `@ts-ignore` for this specific case.

Reply:
Yup that makes sense